SR2021/2021-02-14-league-2-followup.md

```diff
@@ -0,0 +1,34 @@
+---
+to: Teams who attended the leagues but aren't very active
+subject: Preparing for league 3
+---
+
+Hi,
+
+It was great to see your robot at the second league session. We hope you found
+it as exciting as we did and are looking forward to the new challenges which
+module Ⅲ brings. The full rulebook and simulator updates are available on our
+website, so do check them out if you haven't already!
+
+We just wanted to check on how you're doing and whether there's any more help
+you need?
+
+As a reminder, there's usually lots of volunteers around in Discord to help out
+if you need anything, whether it's help programming your robot, organising your
+team or you just need a fresh pair of eyes on something. Our volunteers have a
+wealth of experience from both competing in Student Robotics and helping lots of
+teams, so there's a good chance they can help :)
+
+We also have a limited number of volunteers who would be able to regularly
+mentor your team. This would provide you some more hands-on support in the form
+of a volunteer who is able to be around whenever you regularly meet and would
+get to know your team and your robot. Whatever platform you use (whether it's
+Discord or something else), they'd be able to join your meetings to help you
+out. (For safeguarding reasons we need your team leader to also be in any video
+calls; your team leader will have details about this).
```

Comment:
```suggestion
out. (For safeguarding reasons we need you to also be in any video calls).
```

Reply:
c6bca27 fixes this for all three emails.